code/modules/vehicles/van/box_van.dm

```diff
@@ -0,0 +1,290 @@
+
+//Trucks
+//Read the documentation in multitile.dm before trying to decipher this stuff
+
+/obj/vehicle/multitile/box_van
+	name = "Colony Van"
```

Comment:
Either uncapitalize or make improper:
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/6244c131-fa8f-48e3-8f8e-78db5d50a58d)

Should also fix the same for the wheels too and set their gender to plural https://www.byond.com/docs/ref/#/atom/var/gender

Reply:
The "Van Wheels" are just the default wheels already in the game, in the van_wheels.dm file, should I change it there?

Reply:
Yes